src/libs/ajax/AzureStorage.ts

```diff
@@ -111,6 +111,54 @@ export const AzureStorage = (signal?: AbortSignal) => ({
     }), notebooks)
   },
 
+  blobMetadata: (azureStorageUrl: string) => {
+    const getObjectMetadata = async () => {
+      const workspaceId = azureStorageUrl.split('/')[3].replace('sc-', '')
+      const fileName = _.last(azureStorageUrl?.split('/'))?.split('.')?.join('.')
```

Comment:
`azureStorageUrl` is defined as a string, not `string | undefined`. Is the `?` necessary here?